CONTRIBUTING.md

```diff
@@ -68,3 +78,20 @@ To release any project in this repository:
 5. Tag the above commit with as `<project>/v<version>` with `git tag $(changie latest -j <project>) <commit-sha>`.
     - If the operator is being released, also tag the same commit as `v<version>`.
 6. Push the tag(s).
+
+## Nightly build
+
+### Schedule builds
+
+Buildkite has configured periodic jobs that build and pushes operator container with operator
+helm chart to https://hub.docker.com/r/redpandadata/redpanda-operator-nightly container repository.
+The following branches are currently build periodically:
```

Comment:
Let's not keep a list here as I imagine it'll quickly become out of date? We've got this noted in many other places 😬  You could link to the README or another source for this?

Could you also add a link to the buildkite configuration for the nightly builds? `[SOMETEXT](.buildkite/pipeline.yaml#45)` should do it